rdwatch/core/views/animation.py

```diff
@@ -0,0 +1,218 @@
+import logging
+
+from celery.result import AsyncResult
+from ninja import Router
+from pydantic import UUID4, BaseModel
+
+from django.http import HttpRequest, HttpResponse
+from django.shortcuts import get_object_or_404
+
+from rdwatch.core.models import AnimationModelRunExport, AnimationSiteExport
+from rdwatch.core.tasks.animation_export import (
+    GenerateAnimationSchema,
+    create_animation,
+    create_modelrun_animation_export,
+    create_site_animation_export,
+)
+
+logger = logging.getLogger(__name__)
+
+router = Router()
+
+
+def generate_download_data(
+    exports: list[AnimationSiteExport] | list[AnimationModelRunExport],
+):
```

Comment:
These type annotations are incorrect. To be exact, you're passing `QuerySets` as `exports` where this function is used -
```suggestion
from django.db.models import QuerySet

def generate_download_data(
    exports: QuerySet[AnimationSiteExport] | QuerySet[AnimationModelRunExport],
):
```

Alternatively, this can be typed more abstractly -
```suggestion
from typing import Iterable

def generate_download_data(
    exports: Iterable[AnimationSiteExport] | Iterable[AnimationModelRunExport],
):
```

Reply:
Yup at some point I think I did convert them into a list, but removed it.